ui/common/ResourcePage.tsx

```diff
@@ -46,6 +47,10 @@ export default function ResourcePage({
             <div className="flex flex-col gap-[15px]">{codeResources}</div>
           </>
         )}
+        <>
+        <hr className="my-7 h-[1px] w-full opacity-25" />
+        <Text className="text-3xl" >{t('help_page.feedback')} </Text>
```

Comment:
lets use the footer tag here and make the text quite a bit smaller,  think its too big

Reply:
did you mean the footer component?

Reply:
no i think its fine like this